api/queue/work/model_service_deployment_test.go

```diff
@@ -869,3 +1005,10 @@ func TestExecuteRedeployment(t *testing.T) {
 		})
 	}
 }
+
+func createDefaultMockDeploymentStorage() *mocks.DeploymentStorage {
```

Comment:
Added this helper method here to build a default mocked deployment storage component.